### DISCUSSION TITLE
how to set password for unattended access on the 32 bit version of the app

### DISCUSSION THREAD

**Discussion by Valka-prog** [rustdesk/rustdesk#8217] [General]
Hello,how to set password for unattended access on the 32 bit version of the app it is different from the 64 bit and the pc has a 32 Bit processor so i can't put a 64 bit program?

**Comment by mac1-131** [comment]
click on pencil icon next to one-time password

change that menu to use permanent password and set permanent password

everything is there in x86 version (which I think is sciter build) it just appears differently on the menus

**Reply by Valka-prog** [reply]
Thanks!
